docs/Power-Monitoring-Calibration.md

```diff
@@ -35,16 +35,21 @@
    [`VoltageSet <voltage>`](Commands.md#voltageset)  
    _Replace `<voltage>` with your standard voltage or with reading on your multi-meter if you have one. Your voltage will vary depending on the electrical standards and your electrical grid_  
 
-3. Verify the **Current** reading by calculating current value (amperage) using this formula: **P<sub>(W)</sub>/V<sub>(V)</sub>=I<sub>(A)</sub>**. Adjust the current offset if needed (in milliAmps (mA=A\*1000)):  
+3. Verify the **Current** reading by calculating current value (amperage) using this formula: **P<sub>(W)</sub>/V<sub>(V)</sub>=I<sub>(A)</sub>**. Adjust the current offset if needed (in milliAmps ($1000mA=1A$)):  
    [`CurrentSet <current>`](Commands.md#currentset)  
    _Replace `<current>` with your calculated value (in milliAmps)_  
 
    `CurrentSet` calculation:   
-   P/V=I
-   1000 * Watts/Volts = milliAmperes
+   $$\frac{Watts}{Volts}=Amps$$
+   
+   and thus
+   
+   $$1000 \* \frac{Watts}{Volts} = milliAmps$$
 
 !!! example
-     1000*(60.0/235.5) = 254.777
+     Consider a power usage of $60W$ at a voltage of $240V$. Then the current in milliAmps can be calculated as
+     
+     $$1000 \* \frac{60}{240} = 240.0$$
```

Comment:
According to my calculator is (60 / 240) * 1000 = 250 ?!?

Reply:
You are completely correct, I have changed it to hopefully now be right. Thanks for the comment :)